snapshot/diffapply_linux.go

```diff
@@ -1,6 +1,3 @@
-//go:build !windows
-// +build !windows
-
```

Comment:
Are we sure there's not any "non-linux and non-freebsd" users of this code? ISTR there was some experimental code in containerd to allow it to be built on macOS; just making sure there's not anyone using this on "other" unix-y platforms.

Reply:
Nope. I see references to Darwin here and there throughout the codebase, so it's best to resurrect this file.